tests/www/test_security.py

```diff
@@ -87,7 +87,7 @@ def setUpClass(cls):
         cls.appbuilder = cls.app.appbuilder
         cls.app.config['WTF_CSRF_ENABLED'] = False
         cls.security_manager = cls.appbuilder.sm
-        cls.delete_roles()
+        cls.delete_test_roles()
```

Comment:
Is this only a simple rename or is `delete_roles` not technically doable?

Reply:
Just a rename. Otherwise, it looks like a some important api function.
Rename makes it clear that this is just a test suites related func.

Reply:
Thanks for the clarification.